AaxDecrypter/AaxcTagLibFile.cs

```diff
@@ -0,0 +1,165 @@
+
+        public string Comment => AppleTags.Comment;
+        public string[] Authors => AppleTags.Performers;
+        public string FirstAuthor => Authors?.Length > 0 ? Authors[0] : default;
+        public string TitleSansUnabridged => AppleTags.Title?.Replace(" (Unabridged)", "");
+        public string BookCopyright => _copyright is not null && _copyright.Length > 0 ? _copyright[0] : default;
+        public string RecordingCopyright => _copyright is not null && _copyright.Length > 1 ? _copyright[1] : default;
+        public string Narrator 
+        { 
+            get
+            {
+                string[] text = AppleTags.GetText(narratorType);
+                return text.Length == 0 ? default : text[0];
+            }
+        }
+        public string LongDescription 
+        { 
+            get
+            {
+                string[] text = AppleTags.GetText(descriptionType);
+                return text.Length == 0 ? default : text[0];
+            }
+        }
+        public string ReleaseDate
+        {
+            get
+            {
+                string[] text = AppleTags.GetText("rldt");
+                return text.Length == 0 ? default : text[0];
+            }
+        }
+        public string Publisher
+        {
+            get
+            {
+                string[] text = AppleTags.GetText(publisherType);
+                return text.Length == 0 ? default : text[0];
+            }
+        }
+
+        private string[] _copyright => AppleTags.Copyright?.Replace("&#169;", string.Empty)?.Replace("(P)", string.Empty)?.Split(';');
+        public AaxcTagLibFile(IFileAbstraction abstraction)
+            : base(abstraction, ReadStyle.Average)
+        {          
+        }
+
+        public AaxcTagLibFile(string path) 
+            : this(new LocalFileAbstraction(path))
+        {
+        }
+        /// <summary>
+        /// Copy all metadata fields in the source file, even those that TagLib doesn't
+        /// recognize, to the output file.
+        /// NOTE: Chapters aren't stored in MPEG-4 metadata. They are encoded as a Timed
+        /// Text Stream (MPEG-4 Part 17), so taglib doesn't read or write them.
+        /// </summary>
+        /// <param name="sourceFile">File from which tags will be coppied.</param>
+        public void CopyTagsFrom(AaxcTagLibFile sourceFile)
+        {
+            AppleTags.Clear();
+
+            foreach (var stag in sourceFile.AppleTags)
+            {
+                AppleTags.SetData(stag.BoxType, stag.Children.Cast<AppleDataBox>().ToArray());
+            }
+        }
+        public void AddPicture(byte[] coverArt)
+        {
+            AppleTags.SetData("covr", coverArt, 0);
+        }
+
+        /// <summary>
+        /// Attempts to convert unicode characters to an approximately equal ASCII character.
+        /// </summary>
+        private string unicodeToAscii(string unicodeStr)
```

Comment:
In case it's relevant to your process: I intend to later extract this to Dinah.Core. It's perfect for reuse